### DISCUSSION TITLE
Add block content support

### DISCUSSION THREAD

**Discussion by simovicaleksa** [medusajs/medusa#7368] [Feature Requests]
### ✏ About Feature

Block content is a fairly straight-forward way to create, edit and manage content. It is an industry standard and all bigger stores are using it for their product pages. Shopify has this functionality and it is the biggest advantage compared to other e-commerce solutions.

All products on Amazon and other bigger online shops have a "below the fold" section in product pages, that include additional information regarding the product:

- Images showcasing product quality.
- Tables for product specifications.
- Lists of use-cases for the product.
- Custom elements such as: Newsletter subscriptions, Add to cart sections and more...

From a business point of view this is the best way to increase the conversion rate, since the customer that is interested in the product will naturally try to find more information about the product by scrolling down.

### 📝 CMS Plugins

While it is currently possible to implement this functionality by integrating Contentful or Strapi CMS plugins, it should be much simpler to implement such a basic and standardized functionality of online shops.

Editors would also appreciate not having to learn their way around multiple platforms for adding products and editing the content for products.

### 🚀 Medusa 2.0

With Medusa 2.0 just around the corner, you should look into the opportunity of creating block content a feature. It makes life for developers and editors much easier, while possibly reducing the costs of hosting. This feature is awesome for the marketing campaign of Medusa 2.0 and could bring more users to your awesome headless CMS.

### ❤ Final Words

Thank you for taking your time to read this post, much love to the Medusa team for creating such an awesome platform and hopefully it gets even better in the future. ❤

**Comment by SalahAdDin** [comment]
@Aleksa1312 You can add other points related to branding and advertising like branch banners.
